python/src/lib.rs

```diff
@@ -840,22 +846,22 @@ impl RawDeltaTable {
     pub fn get_py_storage_backend(&self) -> PyResult<filesystem::DeltaFileSystemHandler> {
         Ok(filesystem::DeltaFileSystemHandler {
             inner: self._table.object_store(),
-            rt: Arc::new(rt()?),
+            rt: Arc::new(rt_pyo3()?),
```

Comment:
`DeltaFileSystemHandler` (https://github.com/delta-io/delta-rs/blob/main/python/src/filesystem.rs#L58) uses the `runtime` in utils.rs (https://github.com/delta-io/delta-rs/blob/main/python/src/utils.rs#L10-L13), which is not `pyo3-asyncio`. So I keep the original `rt` but change name to `rt_pyo3` to avoid errors.
Is there a better way to handle this?

Reply:
Interesting, so utils.rs and lib.rs both have a runtime that they create? I don't actually know what is the right approach here without more knowledge of runtimes and tokio. Should we get help or learn ourselves?